pytx3/pytx3/signal_type/raw_text.py

```diff
@@ -24,5 +29,69 @@ class RawTextSignal(signal_base.SimpleSignalType, signal_base.StrMatcher):
     INDICATOR_TYPE = "DEBUG_STRING"
     TYPE_TAG = "media_type_text"
 
+    def __init__(self) -> None:
+        super().__init__()
+        self.normal_to_raw = {}
+
     def match(self, content: str) -> t.List[signal_base.SignalMatch]:
         return self.match_hash(content)
+
+    def match_hash(self, signal_str: str) -> t.List[signal_base.SignalMatch]:
```

Comment:
why separate these functions? If the purpose is it is easy to remove the normalization why not add an optional `normalize : bool` argument?

Reply:
Check out the SignalBase, StrMatcher, and the `match` command to see how match and match_hash are used differently. I'm not sure if the way it's currently used is the best final form of it, but these two methods are separate for a reason.